tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -3494,504 +3493,4 @@ public function testDeleteProject(): array
 
         return $data;
     }
-
-    public function testTenantIsolation(): void
```

Comment:
Are we moving all these tests to cloud now ? Why did we remove them ?

Reply:
Instead of the few single tests, we now have all tests running for both shared and non-shared table modes